PacManAnimation/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en" >
 <head>
   <meta charset="UTF-8">
-  <title>CodePen - pacman svg template literals</title>
+  <title>Pacman</title>
   <link rel="stylesheet" href="./style.css">
```

Comment:
@DragonUncaged what is this

Reply:
I store some of my work in codepen thats why the title was codepen.
As codepen doesn't need simple html body and other basic tag you can code directly

But When you export the file from codepen the codepen automatically adds all the basic tag required with the title codepen - your file name

Reply:
If This is Your Code pen and When You Click On Export.
It Will Automatically Add The Title And HTML tag. 

![image](https://user-images.githubusercontent.com/72241207/167010103-be69588f-c190-46e3-9a35-5619858a3075.png)


You can Try you own.